### PR TITLE
added rhel os check

### PR DESCRIPTION
**Pull request by devidask27**
Added RHEL OS Support. Teated it on RHEL 7.4 and RHEL 7.5
Screenshot 2021-06-25 at 2 51 03 PM
Screenshot 2021-06-25 at 2 51 36 PM

### CODE REVIEW

**Comment by nikhilbandi** [comment]
@devidask27 , have you tested for negative test cases in case of check-node command?
Also, when we display error message in case of unsupported Operating Systems, we should include RHEL (with version numbers) as one of the supported OSes apart from Ubuntu & Centos. Has this been taken care of?

File: pkg/platform/centos/centos.go [hunk 1]
```diff
@@ -273,7 +273,7 @@ func (c *CentOS) Version() (string, error) {
 	out, err := c.exec.RunWithStdout(
 		"bash",
 		"-c",
-		"cat /etc/*release | grep 'CentOS Linux release' -m 1 | cut -f4 -d ' '")
+		"cat /etc/*release | grep 'Red Hat Enterprise Linux Server release' -m 1 | cut -f7 -d ' ' && cat /etc/*release | grep 'CentOS Linux release' -m 1 | cut -f4 -d ' '")
```
Comment (anupbarve) [inline pkg/platform/centos/centos.go:276]: Why are we adding rhel check in centos.go ?

File: pkg/pmk/node.go [hunk 2]
```diff
@@ -241,7 +241,7 @@ func ValidatePlatform(exec cmdexec.Executor) (string, error) {
 	}
 	var platform platform.Platform
 	switch {
-	case strings.Contains(strData, util.Centos) || strings.Contains(strData, util.Redhat):
+	case strings.Contains(strData, util.Centos) || strings.Contains(strData, util.Redhat) || strings.Contains(strData, "Red Hat Enterprise Linux"):
```
Comment (anupbarve) [inline pkg/pmk/node.go:244]: Why is this check needed when we already have Redhat as 2nd check ?

Comment (devidask27) [reply pkg/pmk/node.go]: Removed

**Review by anupbarve — APPROVED** [review]

**Comment by devidask27** [comment]
> @devidask27 , have you tested for negative test cases in case of check-node command?
> Also, when we display error message in case of unsupported Operating Systems, we should include RHEL (with version numbers) as one of the supported OSes apart from Ubuntu & Centos. Has this been taken care of?

Tested for negative test cases in case of check-node
Screenshot 2021-06-29 at 3 51 56 PM
Also checked for port. it is detecting if port is in use.

**Pull request merged**